.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2.1
+
+commands:
+  nuodb_collector_command:
+    steps:
+      - checkout
+      - run:
+          name : Run dependencies
+          command: ./tests/ci/determine_and_run_tests.sh
+      - run:
+          name : Take a nap
+          command: |
+            sleep 15
+      - run:
+          name : Configure and run tests
+          command: |
+            pip install -r test_requirements.txt
+            python ./run_tests.py
+            echo "Done"
```

Comment:
Can you attach the JUnit test reports to CircleCI using `store_test_results` step?

Reply:
Thanks for the suggestion. I have added the test results.